truss-chains/truss_chains/deployment/code_gen.py

```diff
@@ -457,6 +457,19 @@ def _gen_load_src(chainlet_descriptor: definitions.ChainletAPIDescriptor) -> _So
     return _Source(src=src, imports=imports)
 
 
+def _gen_health_check_src(
+    health_check: definitions.HealthCheckAPIDescriptor,
+) -> _Source:
+    def_str = "async def" if health_check and health_check.is_async else "def"
+    maybe_await = "await " if health_check and health_check.is_async else ""
```

Comment:
Seems like we can simplify?
```suggestion
    def_str = "async def" if health_check.is_async else "def"
    maybe_await = "await " if health_check.is_async else ""
```